src/components/Form/Form.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import todosFromServer from '../../api/todos';
+import { User } from '../../interfaces/user';
+import { Todo } from '../../interfaces/todo';
+
+interface Props {
+  users: User[];
+  addTodo: (todo: Todo) => void;
+}
+
+export const Form: React.FC<Props> = ({ users, addTodo }) => {
+  const [title, setTitle] = useState('');
+  const [titleInvalid, setTitleInvalid] = useState(false);
+
+  const [userId, setUserId] = useState(0);
+  const [userIdInvalid, setUserIdInvalid] = useState(false);
+
+  const [id, setId] = useState(Math.max(
+    ...todosFromServer.map(todo => todo.id),
+  ) + 1);
+
+  const fieldChange = (value: string) => {
+    setTitle(value);
+    setTitleInvalid(false);
+  };
+
+  const selectChange = (value: number) => {
+    setUserId(value);
+    setUserIdInvalid(false);
+  };
+
+  const submitForm = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (title.length === 0 || !title.trim()) {
+      setTitleInvalid(true);
+    }
+
+    if (userId === 0) {
+      setUserIdInvalid(true);
+    }
+
+    if (title.length > 0 && userId > 0) {
+      setId(prev => prev + 1);
+
+      const newTodo: Todo = {
+        id,
+        title,
+        completed: false,
+        userId,
```

Comment:
```suggestion
      const newTodo: Todo = {
        id,
        title,
        completed: false,
        userId,
        todoUser: getUserById(userId)
```
and here you can add user immediately during make new todo